statsd/src/main/java/com/avast/metrics/statsd/StatsDMeter.java

```diff
@@ -0,0 +1,41 @@
+package com.avast.metrics.statsd;
+
+import com.avast.metrics.api.Meter;
+import com.timgroup.statsd.StatsDClient;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class StatsDMeter implements Meter {
```

Comment:
Counter holds some value (it's basically `Gauge[Int]` with better API), meter is for measuring rate of events. From StatsD implementation POV, it's almnost the same but you have to derive the data from meter in Grafana.